.github/docker/linux_arm64/Dockerfile

```
@@ -0,0 +1,128 @@
+# Setup the basic necessities
+RUN yum install -y curl zip unzip tar autoconf
+RUN yum install -y ninja-build
+RUN yum install -y perl-IPC-Cmd
+RUN yum install -y ccache
+RUN yum install -y java-11-openjdk-devel maven
+RUN yum install -y wget
+RUN yum install -y kernel-headers
+RUN yum install -y jq
+
+# Install Clang
+RUN apt-get update -qq && \
```

Hadolint (2.12.0)

[info] Delete the apt-get lists after installing something

(DL3009)